### ISSUE TITLE
Transparent background

### ISSUE THREAD

**Issue by maxigaz** [omgold/mantid-term#1]
Is it possible to make the background transparent in mantid? If so, how can I do that?

I've already tried setting the background colour by using an rgba value in `~/.config/mantid.yml`  since that’s how I enabled transparency in Termite with compton.

```yml
appearance:
    ...
    colors:
        ...
        background: rgba(40, 44, 52, 0.96)`
```

**Comment by omgold** [comment]
I have to check why that is not working. If it works in Termite it should be possible to get that, too.

**Comment by omgold** [comment]
Fixed in version 1.0.4.
